todo.txt

```diff
@@ -1,15 +1,7 @@
 - helper log grubs/essence
 - dupe geo exception (in GeoCounter.AddGeo, can't reproduce?)
 - condensed spoiler logs
-- Fix ready list not showing up
-- fix essentially every part of the server because oh god it is so broken
-
-Unhandled Exception: System.NullReferenceException: Object reference not set to an instance of an object.
-   at MultiWorldServer.Server.DoResends(Object clients)
-   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
-   at System.Threading._TimerCallback.PerformTimerCallback(Object state)
-
-- go through modlogs and investigate local crashes related to server
-- go through server logs
-- show list of n recently received items
-- have an eject button that just releases all your items to other players
\ No newline at end of file
+- Fix ready list not showing up (can't reproduce?)
```

Comment:
One method of reproducing:

1. Connect to a MW
2. Ready & Start game
3. Quit to menu
4. New save
5. Connect to MW again
6. Ready